### PR TITLE
A couple changes to increase compatability

### PR DESCRIPTION
**Pull request by YusufKhan-gamedev**
This adds a couple things to increase compatibility a bit on sbase.

### CODE REVIEW

**Review by Geertiebear — APPROVED** [review]
LGTM.

**Comment by ArsenArsen** [comment]
why are those levels macros and where did this code come from?

**Comment by YusufKhan-gamedev** [comment]
Everything was copied from musl

File: options/posix/include/syslog.h [hunk 1]
```diff
@@ -8,6 +8,30 @@
 extern "C" {
 #endif
 
+typedef struct {
+	char *c_name;
+	int c_val;
+} CODE;
+#define prioritynames ((CODE *)(const CODE []){ \
```
Comment (no92) [inline options/posix/include/syslog.h:15]: If these come from musl, putting at least a notice might be necessary, even though this is a header.

**Review by no92 — CHANGES_REQUESTED** [review]
The commit messages are not in the [style they should be](https://docs.managarm.org/handbook/contributing/commit-messages.html).

File: options/posix/include/syslog.h [hunk 2]
```diff
@@ -8,6 +8,30 @@
...
+	{ "auth", LOG_AUTH }, { "authpriv", LOG_AUTHPRIV }, \
+	{ "cron", LOG_CRON }, { "daemon", LOG_DAEMON }, { "ftp", LOG_FTP }, \
+	{ "kern", LOG_KERN }, { "lpr", LOG_LPR }, { "mail", LOG_MAIL }, \
+	{ "mark", INTERNAL_MARK }, { "news", LOG_NEWS }, \
+	{ "security", LOG_AUTH }, { "syslog", LOG_SYSLOG }, \
+	{ "user", LOG_USER }, { "uucp", LOG_UUCP }, \
+	{ "local0", LOG_LOCAL0 }, { "local1", LOG_LOCAL1 }, \
+	{ "local2", LOG_LOCAL2 }, { "local3", LOG_LOCAL3 }, \
+	{ "local4", LOG_LOCAL4 }, { "local5", LOG_LOCAL5 }, \
+	{ "local6", LOG_LOCAL6 }, { "local7", LOG_LOCAL7 }, { 0, -1 } })
```
Comment (ArsenArsen) [inline options/posix/include/syslog.h:31]: why are these not an inline function?

**Review by ArsenArsen — CHANGES_REQUESTED** [review]
and, again, the commits should be reworded (they seem to be separated well enough though)

**Comment by Geertiebear** [comment]
Closing due to inactivity. Feel free to reopen if these changes are still necessary and our comments have been considered.

**Pull request closed without merging**